src/pages/Profile.jsx

```diff
@@ -1,18 +1,56 @@
-import React from 'react';
+import React, {useContext, useEffect, useState} from 'react';
 import { Link } from 'react-router-dom';
+import { AuthContext } from "../context/AuthContext";
+import axios from "axios";
+
 
 function Profile() {
+    const {auth, token} = useContext(AuthContext);
+    const [favColor, setFavColor] = useState('');
+    const [superSecret, setSuperSecret] = useState('');
+
+    function handleHelpClick(){
+        setFavColor(auth.user.color);
+    }
+
+    useEffect(()=>{
+        async function findSecret () {
+            try {
+                const response = await axios.get(`http://localhost:3000/660/private-content`, {
+                    headers: {
+                        "Content-Type": "application/json",
+                        Authorization: `Bearer ${token}`,
+                    }
+                });
+                console.log("response", response);
+              setSuperSecret(response.data.content);
+            } catch (e) {
+                console.log(e);
```

Comment:
Breidt je die nog even uit met een error-state zodat je die kunt communiceren naar de gebruiker in de UI? 😍